lib/PM2ZabbixMonitor.js

```diff
@@ -72,7 +72,11 @@ PM2ZabbixMonitor.prototype._initListeners = function _initListeners() {
  * @returns {string} The item key to send the data to Zabbix with.
  */
 PM2ZabbixMonitor.prototype.getDataKey = function getDataKey(processID, processState, dataItem) {
-	return 'pm2.processes["' + processID + '",' + dataItem + ']';
+    if (processID.match(/[\[\],]/)) {
```

Comment:
* According to the `zabbix_sender` manpage, whitespace also needs quoting.
* It may be cleaner to put the string escaping logic in a separate function, so that `getDataKey` only knows about the prefix and delegates the escaping of strings to somewhere else (after all, it may turn out that `dataItem` must be escaped in the same way).